source/material_model/reactive_fluid_transport.cc

```diff
@@ -247,7 +247,7 @@ namespace aspect
                   // Limit the porosity to be no smaller than 1e-8 when
                   // calculating fluid effects on viscosities.
                   porosity = std::max(porosity,1e-8);
-                  fluid_out->compaction_viscosities[q] = std::max(std::min(out.viscosities[q] * shear_to_bulk_viscosity_ratio * phi_0/porosity, max_compaction_visc), min_compaction_visc);
+                  fluid_out->compaction_viscosities[q] = out.viscosities[q] * shear_to_bulk_viscosity_ratio * phi_0/porosity;
```

Comment:
Is this change supposed to be part of this PR?

Reply:
I think this change should be part of this PR. I initially was moving calculations from melt simple directly into reactive fluid transport, before making new functions for them. I think this change is one instance where I had changed that line to match with the melt simple calculation, then changed it back to the original